js/login.js

```diff
@@ -0,0 +1,49 @@
+import { fetcher } from "./fetcher.js";
+import { LOGIN_API_URL, POSTS_API_URL } from "./common/constants.js";
+import { addToLocalStorage } from "./common/utils/localStorageUtil.js";
+
+const form = document.querySelector("#loginForm");
+const email = document.querySelector("#email");
+const password = document.querySelector("#password");
+
+async function displayPosts() {
+  const posts = await fetcher(POSTS_API_URL, { method: "GET" }, true);
+  console.log(posts);
+  // Pretend to display the posts
+}
+
+async function loginUser(user) {
+  const postBody = JSON.stringify(user);
+  try {
+    const userLoginData = await fetcher(
+      LOGIN_API_URL,
+      {
+        method: "POST",
+        body: postBody,
+      },
+      false
+    );
+    const token = userLoginData.accessToken;
+    addToLocalStorage("accessToken", token);
+
+    window.location.href = "/profile/index.html";
+  } catch (error) {
+    console.error("Login failes", error);
```

Comment:
You need to display a message to the user in the html here, not just log the error. The user will never check the console for the error message.

Very basic example:

`document.querySelector("#message").innerHTML = '<div class="error">Some error message</div>';`

Obviously you would need an element with id="message" in the html.